libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/PhotonUtils.java

```diff
@@ -20,8 +20,9 @@ public static String fixAvatar(final String imageUrl, int avatarSz) {
             return "";
 
         // if this isn't a gravatar image, return as resized photon image url
-        if (!imageUrl.contains("gravatar.com"))
+        if (!imageUrl.contains("gravatar.com")) {
```

Comment:
It's possible the passed imageUrl doesn't exist (for example, when we generate the URL by using [an email hash](https://en.gravatar.com/site/implement/images/)), in which case Gravatar will return the default "mystery man" image. So we want to use Gravatar rather than Photon.